client/src/main/java/org/opensearch/ml/client/MachineLearningClient.java

```diff
@@ -200,13 +200,33 @@ default ActionFuture<SearchResponse> searchModel(SearchRequest searchRequest) {
         return actionFuture;
     }
 
+
     /**
      * For more info on search model, refer: https://opensearch.org/docs/latest/ml-commons-plugin/api/#search-model
      * @param searchRequest searchRequest to search the ML Model
      * @param listener action listener
      */
     void searchModel(SearchRequest searchRequest, ActionListener<SearchResponse> listener);
 
+
+    /**
+     * For more info on search model, refer: https://opensearch.org/docs/latest/ml-commons-plugin/api/#search-model
```

Comment:
change the documentation which is not applicable here. 

Also seems like `searchModel` & `searchModelGroup` both expecting the same parameter and performing the same task. May be we can merge these two functions by `searchItem` or something else?

Reply:
Thanks Dhurbo. Yeah is something we wanted to discuss with the FE team.